worlds/smw/Options.py

```diff
@@ -255,25 +332,33 @@ class MarioPalette(Choice):
     default = 0
 
 
-class ForegroundPaletteShuffle(Toggle):
-    """
-    Whether to shuffle level foreground palettes
-    """
-    display_name = "Foreground Palette Shuffle"
-
-
-class BackgroundPaletteShuffle(Toggle):
+class LevelPaletteShuffle(Choice):
```

Comment:
since you have curated palettes it might be worth considering making this a TextChoice so people can specify using specific curated palettes instead of being forced to get one at random?